addon/-private/environment.js

```diff
@@ -0,0 +1,159 @@
+import Ember from 'ember';
```

Comment:
Just pushed a change to break this up. I'd recommend starting with `environment/index.js` and `create-env-computed.js`, then moving to `array-value.js` and `value.js`